src/python/library/build_wheel.py

```diff
@@ -83,6 +84,12 @@ def sed(pattern, replace, source, dest=None):
         required=False,
         help="Include linux specific artifacts.",
     )
+    parser.add_argument(
```

Comment:
I am wondering how are we dealing with simultaneous specification of both `--linux` and `--windows` flags. As far as I believe we can not cross-compile DLLs and .SOs . Would specifying both the flags are expected to build wheels for both the platforms? 
We should add additional checks on which wheels can be built depending upon the platform of host machine.

Reply:
Good catch! I'll add these to an argparser [mutually exclusive group](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_mutually_exclusive_group).